tests/src/test/scala/actionContainers/ActionContainer.scala

```diff
@@ -104,19 +113,25 @@ object ActionContainer {
       sys.props
         .get("docker.host")
         .orElse(sys.env.get("DOCKER_HOST"))
+        .orElse {
+          // Check if we are running on docker-machine env.
+          Option(WhiskProperties.getProperty("whisk.version.name")).filter(_.toLowerCase.contains("mac")).map {
+            case _ => s"tcp://${WhiskProperties.getMainDockerEndpoint}"
+          }
+        }
```

Comment:
I don't like this, I don't want the runtimes test to rely on WhiskProperties to have a `whisk.properties` file present.
If user is using docker-machine the `DOCKER_HOST` env variable needs to be set and use.